### PR TITLE
CLDR-13812 Add new locales

### PR DESCRIPTION
**Pull request by macchiati**
##### Checklist

- [ ] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-13812
- [ ] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

File: seed/main/nv.xml [hunk 1]
```diff
@@ -17,8 +17,8 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		</orientation>
 	</layout>
 	<characters>
-		<exemplarCharacters references="R1">[a á ą {ą\u0301} b {ch} {ch’} d {dl} {dz} e é ę {ę\u0301} g {gh} h {hw} i í į {į\u0301} j k {kw} {kʼ} l ł m n o ó ǫ {ǫ\u0301} s {sh} t {tł} {tłʼ} {ts} {tsʼ} {tʼ} w x y z {zh} ʼ]</exemplarCharacters>
+		<exemplarCharacters references="R1">[a á ą {ą\u0301} b {ch} {ch‘} d {dl} {dz} e é ę {ę\u0301} g {gh} h {hw} i í į {į\u0301} j k {kw} {kʼ} l ł m n o ó ǫ {ǫ\u0301} s {sh} t {tł} {tłʼ} {ts} {tsʼ} {tʼ} w x y z {zh} ʼ]</exemplarCharacters>
```
Comment (pedberg-icu) [inline seed/main/nv.xml:20]: This seems wrong. You have changed {ch’} using U+2019 to  {ch‘} using U+2018, a different shape and not the one I see in Navajo orthography websites.

Also, all of the following use U+02BC, should they not all be consistent is using U+2019 (or U+2018 if that is really what is intended: ... {kʼ} ... {tłʼ} ... {tsʼ} {tʼ} ... ʼ]

File: seed/main/nv.xml [hunk 2]
```diff
@@ -17,8 +17,8 @@ CLDR data files are interpreted according to the LDML specification (http://unic
...
 		<exemplarCharacters type="numbers">[\- , . % ‰ + 0 1 2 3 4 5 6 7 8 9]</exemplarCharacters>
-		<exemplarCharacters type="punctuation">[\- ‐ – — , ; \: ! ? . … ' ‘ ’ &quot; “ ” ( ) \[ \] § @ * / \&amp; # † ‡ ′ ″]</exemplarCharacters>
+		<exemplarCharacters type="punctuation">[\- ‐ – — , ; \: ! ? . … ' ‘ ‘ &quot; “ ” ( ) \[ \] § @ * / \&amp; # † ‡ ′ ″]</exemplarCharacters>
```
Comment (pedberg-icu) [inline seed/main/nv.xml:22]: Thius is definitenyl wrong, you have changed the paired single quotes U+2018 U+2019 to be U+2018 U+2018

Comment (macchiati) [reply seed/main/nv.xml]: Thanks. The test caught this also; my search and replace was getting the characters confused. Please what it looks like now.

**Comment by macchiati** [comment]
Whoops, need to fix text failure (different cause).

**Pull request merged**